poppy/instrument.py

```diff
@@ -474,8 +474,12 @@ def _getOpticalSystem(self,fft_oversample=2, detector_oversample = None, fov_arc
 
 
         #---- apply pupil intensity and OPD to the optical model
-        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, opdunits='micron', rotation=self._rotation)
+        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, rotation=self._rotation)
```

Comment:
So does this require an immediate update to all the OPD files, or else it will raise the exception about missing units? Or does it handle legacy files semi gracefully in some way?